index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="Style.css">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/>
+        <title>Setup & Mobile Portfolio Skeleton</title>
+</head>
+<body>
+    <main id="root">
+    <header>
+        <nav class="navbar">
+            <a href="#" target="_blank">My Logo</a>
+            <img class="hamburger" src="images/Union.png" alt="">
+            </nav>
```

Comment:
[Optional Change] This change won't stop your project from being approved but it'll help make your it meaningful

- Good job placing your logo in an `a` tag as requested by the previous reviewer. 
I would suggest you place a logo design of your choice or you use your name as the logo, cause this is your portfolio website. Eg can be like `<a href="#" target="_blank">Epaltechs</a>`. What do you think? 👍

Reply:
I really appreciate your advice. I'll do just so. Many thanks.